app/src/pages/inside/launchesPage/modals/launchDeleteModal/launchDeleteModal.jsx

```diff
@@ -0,0 +1,68 @@
+import { Component } from 'react';
+import PropTypes from 'prop-types';
+import Parser from 'html-react-parser';
+import classNames from 'classnames/bind';
+import { injectIntl, defineMessages, intlShape } from 'react-intl';
+import { withModal, ModalLayout } from 'components/main/modal';
+import { COMMON_LOCALE_KEYS } from 'common/constants/localization';
+import styles from './launchDeleteModal.scss';
+
+const cx = classNames.bind(styles);
+
+const messages = defineMessages({
+  deleteLaunchHeader: {
+    id: 'DeleteLaunchDialog.deleteLaunchHeader',
+    defaultMessage: 'Delete launch',
+  },
+  deleteLaunchText: {
+    id: 'DeleteLaunchDialog.deleteLaunch',
+    defaultMessage: "Are you sure to delete launch '<b>{name} #{number}</b>'? It will no longer exist.",
+  },
+});
+
+@withModal('launchDeleteModal')
+@injectIntl
+export class LaunchDeleteModal extends Component {
+  static propTypes = {
+    intl: intlShape.isRequired,
+    data: PropTypes.shape({
+      item: PropTypes.object,
+      onConfirm: PropTypes.func,
+    }),
+  };
+
+  static defaultProps = {
+    data: {
+      item: {},
+      onConfirm: () => {},
+    },
+  };
+
+  render() {
+    const { intl } = this.props;
+    const { item, onConfirm } = this.props.data;
+    const confirmAndClose = (closeModal) => {
```

Comment:
Move to class method so it won't be created on every render

Reply:
fixed